### DISCUSSION TITLE
Cannot Launch WALC using Snap

### DISCUSSION THREAD

**Discussion by jrr45** [cstayyab/WALC#45] [Troubleshooting]
**Describe the bug**
Snap fails to launch and silently exits, no output to the terminal if launched using that. This seems to happen only if the program is terminated by a shutdown without exiting the app first. Reinstalling the snap fixes everything.

**WALC Installation Type:** Snap

**Did you try closing and then opening it back?:** Yes 

**System Information (please complete the following information):**
 - OS/Distro: Kubuntu 20.04

**Comment by daffamumtaz2361** [comment]
Are you sure it exits? Nowhere to be seen in process list? Or is it just tray icon not displayed and you happen to enable Start Hidden option so WALC seems to just vanish?

**Comment by jrr45** [comment]
There isn't anything resembling WALC in system monitor or top from what I can tell. The terminal command exits immediately without seeming to start anything. I haven't changed any settings after reinstalling it.

**Comment by hazard157** [comment]
I confirm the problem with Kubuntu 21.04 and Kubuntu 21.10. Few months ago WALC stared behaving strangely.
First time after installation (via snap) it  starts and connects with QR code. After exit it never starts agail. Both from GUI or command line it silently dies. 

I reproduced it on several PC and laptops. After fresh installation of Kubuntu 21.04 or 21.10 intsall WALC as "span install walc", run it first time - OK, After exitting WALC or rebooting PC WALC never starts again.

I found the only workaround: totally remove ~/snap/walc directory. After it WALC start one time. SO now, every time before I start WALC i must delete ~/snap/walc directory.

Is there any solution or I have to submit bug report like close one [https://github.com/WAClient/WALC/issues/55](url) ?
